x/superfluid/keeper/epoch.go

```diff
@@ -151,7 +151,7 @@ func (k Keeper) UpdateOsmoEquivalentMultipliers(ctx sdk.Context, asset types.Sup
 			return errors.New("osmo should not be a superfluid asset. It can be staked natively")
 		}
 		// get the twap price of the native asset in osmo
-		startTime := k.ek.GetEpochInfo(ctx, k.GetEpochIdentifier(ctx)).StartTime
+		startTime := k.ek.GetEpochInfo(ctx, k.GetEpochIdentifier(ctx)).StartTime // TODO: do 5 mins instead of 1 epoch
```

Comment:
What does this TODO mean? You want the twap over 5 minutes rather than the full epoch? Is there any reason we cant just do that here in this PR?

Reply:
changed